addons/l10n_it/data/template/account.tax.group-it.csv

```diff
@@ -1,12 +1,12 @@
-"id","name","country_id","preceding_subtotal","tax_receivable_account_id","tax_payable_account_id","name@it","pos_receipt_label"
-"tax_group_iva_2","2% VAT","base.it","Imponibile","2605","2605","IVA 2%",""
-"tax_group_iva_4","4% VAT","base.it","Imponibile","2605","2605","IVA 4%","3"
-"tax_group_iva_5","5% VAT","base.it","Imponibile","2605","2605","IVA 5%","4"
-"tax_group_iva_10","10% VAT","base.it","Imponibile","2605","2605","IVA 10%","2"
-"tax_group_iva_12","12% VAT","base.it","Imponibile","2605","2605","IVA 12%",""
-"tax_group_iva_21","21% VAT","base.it","Imponibile","2605","2605","IVA 21%",""
-"tax_group_iva_20","20% VAT","base.it","Imponibile","2605","2605","IVA 20%",""
-"tax_group_iva_22","22% VAT","base.it","Imponibile","2605","2605","IVA 22%","1"
-"tax_group_imp_esc_art_15","VAT Excluded Art.15","base.it","Imponibile","2605","2605","Imponibile Escluso Art.15",""
-"tax_group_fuori","VAT-free","base.it","Imponibile","2605","2605","Fuori Campo IVA","10"
-"tax_group_split_payment","Split Payment","base.it","Scissione dei Pagamenti Esclusa","2607","2607","Scissione dei Pagamenti",""
+"id","name","country_id","preceding_subtotal","tax_receivable_account_id","tax_payable_account_id","name@it","pos_receipt_label","preceding_subtotal@it"
```

Comment:
```suggestion
"id","name","country_id","preceding_subtotal","tax_receivable_account_id","tax_payable_account_id","pos_receipt_label","name@it","preceding_subtotal@it"
```
Just so we try and keep some sort of "order", could you please keep all the translations together and make sure they begin after the "main" columns?
Maybe have a dedicated commit, before this one, to move `name@it` since it's unrelated?